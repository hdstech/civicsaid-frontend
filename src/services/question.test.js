import "isomorphic-fetch";

test('should return a single question object', async () => {
	try {
		const response = await fetch(`http://civicsaid-backend.dev/questions/show-question/6`);
		const fetchedQuestion = await response.json();
		expect(fetchedQuestion).toMatchSnapshot();
	} catch (e) {
		fail();
	}
});
