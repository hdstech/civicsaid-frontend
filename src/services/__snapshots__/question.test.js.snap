// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should return a single question object 1`] = `
Object {
  "question": Object {
    "category": "AMERICAN GOVERNMENT",
    "created_at": "2017-03-04 18:22:42",
    "id": 6,
    "is_favorite": 0,
    "q_chinese": "列舉憲法第一條修正案中的一項權利或自由 。*",
    "q_english": "What is one right or freedom from the First Amendment?*",
    "q_spanish": "¿Cuál es un derecho o libertad que la Primera Enmienda garantiza?*",
    "subcategory": "Principles of American Democracy",
    "updated_at": "2017-03-04 18:22:42",
  },
}
`;
